### PR TITLE
enh(ha)rewrite-architecture-introduction

### PR DESCRIPTION
**Pull request by Sims24**
## Description

Let's rewrite the Centreon HA introduction to describe:
- what do we support 
- main concepts 

Schema and architecture update included

edit: I'll add the FR part after validation of the content. 

## Target serie

- [ ] 20.04.x
- [ ] 20.10.x
- [X] 21.04.x (master)

### CODE REVIEW

File: en/installation/installation-of-centreon-ha/overview.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+For example, Centreon-HA uses location constraints to specify to Pacemaker that the database process 
+needs to be up on backend nodes but not on frontend nodes. 
+
+Regarding colocation constraints, they can ensure a Virtual IP sticks to the master nodes and/or role. 
+Therefore Users, Pollers, and Daemons constantly interact with the primary node. 
+
+### QDevice and votes
+
+The configuration of a qdevice is **mandatory** to avoid split-brain and other situations nobody wants 
+to face in a Cluster. The server with the `quorum-device` role aims to obtain an absolute majority in a vote
```
Comment (pkriko) [inline en/installation/installation-of-centreon-ha/overview.md:93]:
```suggestion
to face in a Cluster. The server with the `quorum-device role aims` to obtain an absolute majority in a vote 
```

File: en/installation/installation-of-centreon-ha/overview.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+Schemas below show both architecture flavor and network flows between servers. To get the complete network 
+flow matrix, refer to the architecture dedicated installation page.
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--Two-nodes-cluster-->
+
+![image](../../assets/integrations/centreon-ha/centreon-ha-2-nodes-arch.png)
+
+Reach [this page](../../installation/installation-of-centreon-ha/installation-2-nodes.png) to start your two nodes setup!
```
Comment (pkriko) [inline en/installation/installation-of-centreon-ha/overview.md:134]:
```suggestion
Reach [this page](../../installation/installation-of-centreon-ha/installation-2-nodes.html) to start your two nodes setup! 
```

File: en/installation/installation-of-centreon-ha/overview.md [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+
+![image](../../assets/integrations/centreon-ha/centreon-ha-2-nodes-arch.png)
+
+Reach [this page](../../installation/installation-of-centreon-ha/installation-2-nodes.png) to start your two nodes setup! 
+
+<!--Four-nodes-cluster-->
+
+![image](../../assets/integrations/centreon-ha/centreon-ha-4-nodes-arch.png)
+
+Reach [this page](../../installation/installation-of-centreon-ha/installation-4-nodes.png) to start your two nodes setup!
```
Comment (pkriko) [inline en/installation/installation-of-centreon-ha/overview.md:140]:
```suggestion
Reach [this page](../../installation/installation-of-centreon-ha/installation-4-nodes.html) to start your four nodes setup!
```

File: en/installation/installation-of-centreon-ha/overview.md [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+
+## Additionnal information
+
+### Server organization
+
+Setting up a Centreon-HA cluster might be overkill or at least not optimal when all your servers are running in 
+the same datacenter or the same rack. 
+
+In a perfect world, the primary and secondary nodes are running on different (geographical) sites, and the qdevice 
+communicate with both sites independently.
```
Comment (pkriko) [inline en/installation/installation-of-centreon-ha/overview.md:150]: Do we need to be precise that all the cluster servers (device include) must be running on the same network?

File: en/installation/installation-of-centreon-ha/overview.md [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+
+### VIP and load balancing
+
+Centreon recommends using VIP addresses.
+
+Use a load balancer is an option but it should support custom rules to route application flows.
+
+For example, in a four nodes setup, a load balancer can rely on:
+* the listening port or the apache process state to route Users and Pollers' communication toward frontend servers.
+* the value of the "read_only" flag on both database servers to determine which one is the primary one.
```
Comment (pkriko) [inline en/installation/installation-of-centreon-ha/overview.md:167]: Precise that we speak about 2 differents VIPs here?

**Review by pkriko — CHANGES_REQUESTED** [review]

File: en/installation/installation-of-centreon-ha/overview.md (hunk 1 above)
Comment (Sims24) [reply en/installation/installation-of-centreon-ha/overview.md:93]: Hum, I'm not sure why this modification is required?

File: en/installation/installation-of-centreon-ha/overview.md (hunk 4 above)
Comment (Sims24) [reply en/installation/installation-of-centreon-ha/overview.md:150]: Ok, I add stg

**Review by pkriko — APPROVED** [review]

**Pull request merged**